src/Controller/BlogController.php

```diff
@@ -46,9 +47,13 @@ class BlogController extends AbstractController
      * Content-Type header for the response.
      * See https://symfony.com/doc/current/quick_tour/the_controller.html#using-formats
      */
-    public function index(int $page, string $_format, PostRepository $posts): Response
+    public function index(Request $request, int $page, string $_format, PostRepository $posts, TagRepository $tags): Response
```

Comment:
why do you call the repository $tags instead of $repository?

Reply:
`$repository` is too broad ... is it a tag repository, a post repository, a user repository? It should be `$tagRepository` .... but we decided to try using this way of naming vars related to repository: instead of `$fooInSingularRepository` we call it `$fooInPlural`. This decision is not definitive. We're still thinking about this (by the way, other frameworks do this too).

Reply:
tagRepository is perfect for me
but if we rename $tags, we need to rename $posts